Makefile

```diff
@@ -74,6 +74,9 @@ build-resource-processor-vm-porter-image:
 build-gitea-image:
 	$(call build_image,"gitea","templates/shared_services/gitea/version.txt","templates/shared_services/gitea/Dockerfile","templates/shared_services/gitea/")
 
+build-gitea-workspace-service-image:
```

Comment:
Do we need to add these new targets at the top so that it is part of `make images`?

Reply:
Added to docs. And created issue to discuss: https://github.com/microsoft/AzureTRE/issues/1472